udplog/test/test_kafka.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Ralph Meijer.
+# See LICENSE for details.
+
+"""
+Tests for L{udplog.redis}.
```

Comment:
Actually, you *can* call it `kafka.py` because we use absolute imports everywhere in this library (`from __future__ import absolute_import`).